day_6/exercises/burrito.js

```diff
@@ -16,12 +16,25 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping() {
+    console.log(this.toppings + " peppers")
+  }
+
+  removeTopping() {
+    console.log(this.toppings)
+  }
+
+  changeProtein() {
+    console.log(this.protein = "chicken")
```

Comment:
This is the closest one... How might you make this dynamic to allow for a change specified when the method is called?

Reply:
So instead of the protein always changing to chicken, it could be changed to whatever is passed in?